.github/workflows/release.yml

```diff
@@ -36,11 +37,49 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
+      # Ensure npm 11.5.1 or later is installed
+      - name: Install latest npm
+        run: npm install -g npm@latest
+
       - name: Release
         uses: changesets/action@v1
         with:
           publish: pnpm release
           version: pnpm changeset-version
         env:
           GITHUB_TOKEN: ${{ secrets.SEEK_OSS_CI_GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  snapshot:
+    name: Publish snapshot version
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@main
```

Comment:
`@main` is interesting 🤔, and I wonder if we want to set up Renovate/Dependabot for the other actions

Reply:
Hmmmm might circle back around once I've merged these many PRs